### PR TITLE
GCI TASK (Add a new potion to the Potions module)(DhananjayGarg19)

### PR DESCRIPTION
**Pull request by dhanjo**

### CODE REVIEW

**Comment by Cervator** [comment]
Hi @DhananjayGarg19! I am afraid this potion won't work, "Die" is not a registered effect and if it was I expect it should outright kill the player, not just subtract 10 HP.  In fact there is [an existing HarmPotion](https://github.com/Terasology/Potions/blob/master/assets/prefabs/potion/HarmPotion.prefab) that does that exact thing (just 4 HP) and works as it uses the valid (and well named) "HARM" effect.

Some suggestions:

* Make sure you set up to run from code properly so you can test the effects of a potion in-game
* Look at the available effects in `PotionCommonEffects` in the Potion module to get some inspiration for what can be done
* Consider whether you'd want to take on the challenge of writing an entirely new effect - we have some good examples at this point.

At a minimum I suggest correcting the effect name and increasing the magnitude to where it is more likely to kill the user. In that case you may also want to rename the potion to something like "DeathPotion" instead

File: assets/prefabs/potion/DeathPotion.prefab [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "DisplayName": {
+        "name": "Death Potion",
+        "description": "Harms 100 HP."
+    },
+    "Potion": {
+        "hasGenome": false,
+        "effects": [
+            {
+                "effect": "Death",
```
Comment (Avalancs) [inline assets/prefabs/potion/DeathPotion.prefab]: "Death" is not an existing effect, you still need to find the effect that harms the player in `PotionCommonEffects`.

File: assets/prefabs/potion/DeathPotion.prefab [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Potions:DeathPotionBottle",
```
Comment (Avalancs) [inline assets/prefabs/potion/DeathPotion.prefab]: For this to work, you need to rename the picture to DeathPotionBottle.png

**Review by Avalancs — CHANGES_REQUESTED** [review]
Please look at the two comments I added.

Also the picture you attached has a white background, which will not be transparent inside the game. We will make a wiki page on how to turn the background transparent soon.

**Comment by Cervator** [comment]
Merged in https://github.com/Terasology/Potions/commit/6c4fe02a08e2bcec85ca4bc6dfa577f887476e9c with some fixes on top in https://github.com/Terasology/Potions/commit/ed234145f91398caabfedad914a05e2d1c9562d1 - closing as complete

**Pull request closed without merging**